Source/drlg_l1.cpp

```diff
@@ -2201,6 +2184,7 @@ void GenerateLevel(lvl_entry entry)
 	}
 
 	bool doneflag;
+	uint8_t floorID = 13;

```

Comment:
Not much point in having a local variable for this when it just gets pass to another function

Reply:
Updated the pull request to remove local variable floorID in all three places. Thank you @AJenbo !

Reply:
I was too slow 😅 . You can ignore that part of my comments.